docs/models/blockstates/introduction.md

```diff
@@ -20,13 +20,8 @@ As an example, let's take a look at the vanilla `oak_log.json`:
 
 As you can see there is no normal state, only different variants depending on the value of "axis". Depending on which axis the log is aligned it will use either a model of the upright log, a model of the sideways log (rotated by 90° or not) or, should the block not have any axis set, it'll display the bark model which has the bark on all 6 sides.
 
-The log only has one property: axis. A blockstate always has to be defined for all of its properties. This can quickly lead to a combinatorial explosion of variants. Let's look at one variant of the vanilla fence:
+The log only has one property: axis. A blockstate always has to be defined for all of its properties. In 1.8, you had to specify the entire variant string, for each and every variant string. This would lead to a very large number of variants, as each and every combination of properties must be defined. The [Forge blockstate format][Forge blockstate] allows you to keep this under control. 1.9 also introduces the "multipart" blockstate format, which has certain advantages and disadvantages compared to Forge's. You can find a description of 1.9's format on the [wiki].
```

Comment:
The 1.9 sentence should start with "Minecraft 1.9". It also needs a bit of a rewording along the lines of explaining that both blockstate types allow for similar things (multiparts) but each of them has specific cases that the other one doesn't cover. Also that neither is better/worse, and that forges blockstates are supported from 1.8 on through 1.10+ and don't end in 1.9.